src/openforms/forms/api/viewsets.py

```diff
@@ -83,12 +92,36 @@ class FormLogicViewSet(
 ):
     serializer_class = FormLogicSerializer
     queryset = FormLogic.objects.all()
-    permission_classes = [permissions.IsAdminUser]
+    permission_classes = [
+        permissions.IsAdminUser
+    ]  # TODO: connect with can_change_form admin permission!
```

Comment:
Todo for later?

Reply:
yes, Joeri and I are aware of this and it's tracked _elsewhere_ :wink: